src/Components/SingleReview.jsx

```diff
@@ -1,6 +1,18 @@
-
+import { useEffect, useState } from "react";
+import { patchVote } from "../util/api";
 
 const SingleReviewTab=({review})=>{
+const [votes,setVotes]=useState(review.votes);
+
+const updateVotes=(value,id,target)=>{
+  value > 0 ? setVotes((votes)=>votes+1) : setVotes((votes)=>votes-1)
+  patchVote(value,id).then(({updatedReview})=>{
+  }).catch((err)=>{
+    value > 0 ? setVotes((votes)=>votes-1) : setVotes((votes)=>votes+1)
+    alert(`There was something wrong with the data! Try again!`)
+  })
+target.disabled=true;
+}
```

Comment:
This means you can only click upvote and downvote both once but if you upvoted, and then downvoted, you should be able to vote back up. You either want to give an upvote OR a downvote on something. Try to have some behaviour that enables the upvote button again, if we click the downvote

Reply:
Also currently the behaviour of your downvote button, if you have already upvoted, just removes the upvote and doesnt actually down vote